src/extension.ts

```diff
@@ -66,6 +66,7 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
 
         ext.fileSystem = new DatabasesFileSystem(ext.tree);
         context.subscriptions.push(vscode.workspace.registerFileSystemProvider(DatabasesFileSystem.scheme, ext.fileSystem));
+        vscode.workspace.getConfiguration().update(ext.settingsKeys.vsCode.autoSave, "off");
```

Comment:
We don't want to just turn this setting off without alerting a user.  Also, I believe we only want to change the setting if the user is running in Codespaces.

I think we should check to see the platform, and then alert the user that we changed the setting because it will cause problems to leave it on in Codespaces.